src/Provider/GoogleMaps/Model/GoogleAddress.php

```diff
@@ -483,15 +483,15 @@ public function getSubLocalityLevels()
      *
      * @return $this
      */
-    public function withSubLocalityLevels(array $subLocalityLevel)
+    public function withSubLocalityLevels(array $subLocalityLevel): GoogleAddress
     {
         $subLocalityLevels = [];
         foreach ($subLocalityLevel as $level) {
             if (empty($level['level'])) {
                 continue;
             }
 
-            $name = $level['name'] ?? $level['code'] ?? null;
+            $name = $level['name'] ?? $level['code'] ?? '';
```

Comment:
Why was this line changed?

Reply:
new AdminLevel() $name parameter requires a string. Its never called because of the empty check before, but the given name parameter should be a string. 
It doesnt matter, but my editor shows it as wrong.

Reply:
Great. Thank you.